### ISSUE TITLE
xRDSessionCollectionConfiguration: Various Parameters are not properly compared

### ISSUE THREAD

**Issue by jeremyciak** [dsccommunity/xRemoteDesktopSessionHost#82]
#### Details of the scenario you tried and the problem that is occurring
Supplying values for certain parameters either does not function as intended, or sometimes even throws errors. Parameters impacted:
- `CustomRdpProperty`
- `ExcludeFilePath`
- `ExcludeFolderPath`
- `IncludeFilePath`
- `IncludeFolderPath`

#### Suggested solution to the issue
Perform some normalization against the provided parameters and current values to do a proper comparison

#### The DSC configuration that is used to reproduce the issue (as detailed as possible)
```powershell
xRDSessionCollectionConfiguration 'TestSessionCollection'
{
    CollectionName        = 'TestSessionCollection'
    EnableUserProfileDisk = $true
    DiskPath              = 'C:\temp'
    ExcludeFilePath       = @()
    ExcludeFolderPath     = @('C:\temp\foo', 'C:\temp\bar')
    IncludeFilePath       = @()
    IncludeFolderPath     = @()
}
```

#### The operating system the target node is running
```powershell
OsName               : Microsoft Windows Server 2019 Datacenter
OsOperatingSystemSKU : DatacenterServerEdition
OsArchitecture       : 64-bit
WindowsVersion       : 1809
WindowsBuildLabEx    : 17763.1.amd64fre.rs5_release.180914-1434
OsLanguage           : en-US
OsMuiLanguages       : {en-US}
```

#### Version and build of PowerShell the target node is running
```powershell
Name                           Value                                                                                                                                                                                                
----                           -----                                                                                                                                                                                                
PSVersion                      5.1.17763.1007                                                                                                                                                                                       
PSEdition                      Desktop                                                                                                                                                                                              
PSCompatibleVersions           {1.0, 2.0, 3.0, 4.0...}                                                                                                                                                                              
BuildVersion                   10.0.17763.1007                                                                                                                                                                                      
CLRVersion                     4.0.30319.42000                                                                                                                                                                                      
WSManStackVersion              3.0                                                                                                                                                                                                  
PSRemotingProtocolVersion      2.3                                                                                                                                                                                                  
SerializationVersion           1.1.0.1
```

#### Version of the DSC module that was used
2.0.0-preview0003

**Comment by jeremyciak** [comment]
PR for proposed solution: #83

**Comment by jeremyciak** [comment]
@johlju or any other contributors... Any thoughts?

**Comment by johlju** [comment]
@ ld0614 do you have time to look at this and the PR proposal?

**Comment by jeremyciak** [comment]
I think you put separation between your '@' and the username. Here is another attempt:
@ld0614

**Comment by danielboth** [comment]
This was fixed in #83
